keras/layers/convolutional.py

```diff
@@ -1449,7 +1464,17 @@ class ZeroPadding2D(Layer):
     # Arguments
         padding: tuple of int (length 2)
             How many zeros to add at the beginning and end of
-            the 2 padding dimensions (axis 3 and 4).
+            the 2 padding dimensions (rows and cols) symmetrically.
+            Setting padding to the values different from (1, 1)
+            overrides top_pad, bottom_pad; and left_pad, right_pad arguments.
```

Comment:
@Itaydal I've took a look at your code and changed some docstrings accordingly.
Regarding ordering, can you elaborate?
I use (samples, channels, rows, cols) ordering for 'th' and (samples, rows, cols, channels) for 'tf'.
I use top_pad and bottom_pad for rows dimension and left_pad and right_pad for cols dimension.